jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -195,6 +198,15 @@ public final class System {
 			osEncoding = definedOSEncoding;
 		}
 		defaultTmpDir = getSysPropBeforePropertiesInitialized(sysPropID_DefaultTmpDir);
+
+		/*[IF (JAVA_SPEC_VERSION >= 21) & (PLATFORM-mz31 | PLATFORM-mz64)]*/
+		/* As part of better handling of JEP400 constraints on z/OS, the com.ibm.autocvt property
+		 * determines whether we convert input file I/O based on file tagging. If not explicitly specified,
+		 * the property defaults to true, unless file.encoding is set to COMPAT.
+		 */
+		zOSAutoConvert = getSysPropBeforePropertiesInitialized(sysPropID_zOSAutoConvert);
+		/*[ENDIF] (JAVA_SPEC_VERSION >= 21) & (PLATFORM-mz31 | PLATFORM-mz64) */
+
```

Comment:
Please don't add this blank line.

Reply:
Thx. Deleted the blank line: [9c68f5c](https://github.com/eclipse-openj9/openj9/pull/19772/commits/9c68f5caec292c7502c6c26dfdfaa201282b060a)